### DISCUSSION TITLE
Reading temperature w/o Building Automation HAT

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#464] [App and O/S Config Questions] [answered]
I have an 10k Temperature probe that I used to connect to an Arduino for the pool temperature.  It is inline on the pipe going into my sta-rite heater.  Do I need to spend $90 for the building automation hat?  Or can I wire it into GPIO?  Do I need a resistor?  Can the system pickup and display the temperature on the dash panel?
Thanks

**Comment by tagyoureit** [comment, marked as answer]
GPIO's are digital inputs and can't read resistance.  You can get a cheaper ADS1115 and hook that up through I2C and REM can use that to feed the temp to njsPC.  https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#sequent-microsystems-building-automation-card

**Comment by rstrouse** [comment]
Here is another wiki with some temp sensor options.
https://github.com/rstrouse/relayEquipmentManager/wiki/Temperature-Sensors
